TypeInjections/TypeInjections/Typecart.fs

```diff
@@ -80,7 +81,7 @@ module Typecart =
         new(d: DirectoryInfo, ignorePatternsFile: string option) = TypecartProject(Utils.D d, ignorePatternsFile)
         // generic entrypoint when we don't know whether path is a file or directory
         new(path: string, ignorePatterns: string option) = TypecartProject(Utils.parseSystemPath(path), ignorePatterns)
-                
+        new(path: string, ignorePatterns: string list) = TypecartProject(Utils.parseSystemPath(path), List.map makeRegex ignorePatterns)
```

Comment:
Is this alright @ruijiefang? I didn't see any constructor with a string list argument so I just added one

Reply:
This is alright. I'm leaving tonight so will just approve this one to unblock you. All the best on your remaining time at AWS!